prisma/data.json

```diff
@@ -3,19 +3,21 @@
         "appId": "CE2679C5-1078-4820-814D-E0D0310A30D9",
         "appIcon": "https://app.scriptoria.io/api/products/CE2679C5-1078-4820-814D-E0D0310A30D9/files/published/app-icon.png",
         "googlePlayUrl": "https://play.google.com/store/apps/details?id=org.example.test",
-        "listing": [
-            {
-                "lang": "en-US",
-                "title": "The App Name From Google Play",
-                "shortDescription": "The Short Description from Google Play",
-                "fullDescription": "The Full Description from Google Play"
-            },
-            {
-                "lang": "es-419",
-                "title": "El nombre de la aplicación de Google Play",
-                "shortDescription": "La breve descripción de Google Play",
-                "fullDescription": "La descripción completa de Google Play"
-            }
-        ]
+        "listings": {
+            "create": [
```

Comment:
@mla04762 This data.json should not have the "create" element around the listings since the code in seed.ts has it already.

Reply:
The only change you need in this file is "listing" to "listings"